test/system/700-play.bats

```diff
@@ -356,3 +355,21 @@ status: {}
     run_podman stop -a -t 0
     run_podman pod rm -t 0 -f test_pod
 }
+
+@test "podman kube down" {
```

Comment:
Apologies for noticing so late.  I think that we should _not_ add new test here but rather add `kube down` to the tests above which would otherwise just `podman stop -a` etc.  That will spare a couple of test cycles compared to adding an entirely new test.